### PR TITLE
Move Traveler from memory to data property

### PR DESCRIPTION
**Pull request by Spedwards**
Before traveler we weren't using the memory property.

Not a fan of inconsistencies.

### CODE REVIEW

**Comment by chobobobo** [comment]
Also not a fan, this will depend on @karlthepagan though as he wants to minimize changes to traveler proper.

**Comment by karlthepagan** [comment]
The location of Traveler.js storage is a contract of that API. Users who migrate from an OCS branch to another AI should respect each other's pathing information if it is available.

I am willing to entertain an accessor function to make integration for visuals simpler, but not a change in Traveler.js functionality.

File: visuals.js [hunk 1]
```diff
@@ -476,13 +476,13 @@ module.exports = class Visuals {
     static drawCreepPath(room) {
         const vis = new RoomVisual(room.name);
         room.creeps.forEach(creep => {
-            if (creep.memory && creep.memory._travel && creep.memory._travel.path) {
```
Comment (karlthepagan) [inline visuals.js]: I would prefer an accessor utility method instead of this refactor.

**Comment by chobobobo** [comment]
@Spedwards do you intend to resolve these changes for the 0.7.0 RC?

**Pull request closed without merging**